scripts/inactive_issue_checker.py

```diff
@@ -137,14 +137,16 @@ def get_open_issues(self) -> List[Issue]:
             requests.HTTPError. Raised if the request fails.
         """
         search_url = 'https://api.github.com/search/issues'
-        url = f'{search_url}?q=repo:oppia/oppia+is:issue+state:open'
+        end = 'is:issue+state:open'
+        url = f'{search_url}?q=repo:{self.repo_owner}/{self.repo_name}+{end}'
```

Comment:
```suggestion
        url = f'{search_url}?q=repo:{self.repo_owner}/{self.repo_name}+is:issue+state:open'
```

why not just include `end` in the `url` template? Unlike `search_url`, I don't see much readability benefit to using an intermediate variable

Reply:
I was getting lint errors, so used a short name. Your suggestion will throw a lint error. Should I disable lint then?